### DISCUSSION TITLE
I2T/T2I Retrieval Performance Increase when I fine-tune the model with CC3M dataset

### DISCUSSION THREAD

**Discussion by hmc0105** [mlfoundations/open_clip#1108] [Q&A]
Hello. I just find that when I finetune the model with CC3M (used LoRA),

I can found that I2T, T2I retrieval performance of other test datasets like coco, nasa, kream etc. increased at initial phase of training.

It seems weird because cc3m is not similar with such test datasets but it happens often.

Does anybody know the reason of this phenomenon?

Thank you.

<img width="260" height="288" alt="Screenshot 2025-08-29 at 2 14 02 PM" src="https://github.com/user-attachments/assets/a7f081cf-617b-419e-935c-58dfa149c8a0" />

[test datasets from huggingface]
visual-layer/oxford-iiit-pet-vl-enriched
hahminlew/kream-product-blip-captions
nkasmanoff/nasa_earth_instagram
lmms-lab/COCO-Caption2017

[config]

model_name: "ViT-B-32"
pretrained: "datacomp_xl_s13b_b90k"
loss : InfoNCE
lora_rank: 16
lora_alpha: 32
lora_dropout: 0.0
batch_size: 32
learning_rate: 1e-5
epochs: 10
warmup: 0.1
